test/Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests/executor/process/ProcessExecutor.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Kafka.LangEndToEndTests.executor.process
+{
+    public class ProcessExecutor : IExecutor<string, Process>
+    {
+        public ProcessExecutor() { }
+
+        public async Task<Process> ExecuteAsync(string request)
+        {
+            Process process = new Process();
+            string shell = null;
```

Comment:
most of the code in this method is for creating the Process, that part could be refactored at least in function for better code readability

Reply:
Done.